### PR TITLE
Feature/delete

### PR DESCRIPTION
**Pull request by priscanicktar**
For efficiently working together

### CODE REVIEW

File: src/server/server.js [hunk 1]
```diff
@@ -23,6 +24,8 @@ app.use(webpackDevMiddleware(compiler, {
 app.get('/', (req, res) => res.send('Hello World!'));
 
 app.get('/api/tasks', (req, res) => new TaskManager(req, res).find());
-app.get('/api/tasks/:id', (req, res) => new TaskManager(req, res).findById(req.params.id));
+app.get('/api/task/:id', (req, res) => new TaskManager(req, res).findById(req.params.id));
+app.delete('/api/task/:id', (req, res) => new TaskManager(req, res).deleteTask(req.params.id));
+app.put('/api/task/:id', (req, res) => new TaskManager(req, res).updateRecords());
```
Comment (edwinmp) [inline src/server/server.js]: Why is this here? As far as I can tell, there's no `updateRecords` function in the task manager.

**Review by edwinmp — CHANGES_REQUESTED** [review]

File: src/server/server.js (hunk 1 above)
Comment (priscanicktar) [reply src/server/server.js]: sure, lemi reslove it

Comment (priscanicktar) [reply src/server/server.js]: sure, lemi resolve it

**Comment by priscanicktar** [comment]
changes made

**Review by edwinmp — APPROVED** [review]

**Pull request merged**